airbyte-integrations/connectors/source-amplitude/source_amplitude/components.py

```diff
@@ -34,7 +34,7 @@ def extract_records(self, response: requests.Response) -> List[Record]:
             series = response_data.get("series", [])
             if len(series) > 0:
                 series = series[0]  # get the nested list
-                return [{"date": date, "length": length} for date, length in zip(response_data["xValues"], series)]
+                return [{"date": date, "length": length["value"]} for date, length in zip(response_data["xValues"], series)]
```

Comment:
Why doesn't the change from using `length` to the length's `value` key result in a change to expected records? I'm a little confused where this value key comes from based on the API docs: https://www.docs.developers.amplitude.com/analytics/apis/dashboard-rest-api/#response_3

Reply:
Synced with Maxime, Amplitude response has also changed (but not documented) that has changed what was original an array of numbers:
```
[12.0, 15.0]
```

to an array of objects:
```
[{ "setId": "", "value": 12.0 }, { "setId": "", "value": 15.0 }]
```

We need this to retain compatibility with our existing connector behavior so this is all good.